Sources/Controls/UIScrollView+Combine.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UIScrollView+Combine.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 09.08.19.
+//
+
+import Foundation
+import Combine
+
+public extension UIScrollView {
+    /// A publisher emitting content offset changes from this UIScrollView.
+    var contentOffsetPublisher: AnyPublisher<CGPoint, Never> {
+        publisher(for: \.contentOffset)
+            .eraseToAnyPublisher()
+    }
+
+    /// A publisher emitting if the bottom of the UIScrollView is reached.
+    ///
+    /// - parameter offset: A threshold indicating how close to the bottom of the UIScrollView.
+    ///                     Default 0.0.
+    /// - returns: AnyPublisher that emits when the bottom of the UIScrollView is reached.
+    func reachedBottomPublisher(offset: CGFloat = 0.0) -> AnyPublisher<Void, Never> {
```

Comment:
```suggestion
    func reachedBottomPublisher(offset: CGFloat = 0) -> AnyPublisher<Void, Never> {
```

No reason for the explicity `0.0` here